integration/monai/monai_nvflare/nvflare_stats_handler.py

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import warnings
+from collections.abc import Callable, Sequence
+from typing import TYPE_CHECKING, Any
+
+import torch
+from monai.config import IgniteInfo
+from monai.utils import is_scalar, min_version, optional_import
+
+from nvflare.apis.analytix import AnalyticsDataType
+from nvflare.app_common.tracking.log_writer import LogWriter
```

Comment:
MONAI should use `LogWriterForMetricExchanger` right?
Because we don't want to pass the whole engine and FLContext object to MONAI ClientAlgo side.